OpenCore-Boot.sh

```diff
@@ -30,7 +30,7 @@ OVMF_DIR="."
 
 # shellcheck disable=SC2054
 args=(
-  -enable-kvm -m "$ALLOCATED_RAM" -cpu Penryn,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,"$MY_OPTIONS"
+  -enable-kvm -m "$ALLOCATED_RAM" -cpu host,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,"$MY_OPTIONS"
```

Comment:
This is a major change. When `-cpu Penryn` is working and has been working for a long time, why should we change to `-cpu host`?

The `-cpu host` doesn't work in 100% of the use cases. Who will maintain and keep maintaining these cpuid related kernel patches?

Reply:
I'm just a bystander (thanks for the repo, btw) but I will say that I can get hardware acceleration working via the hypervisor framework on my OSX-KVM with host but not with Penryn and the suggested cpu bits

I would not advocate for putting something fragile by default in, but I can't be the only user very interested in the hypervisor framework working, so it may be very welcome to have a specific callout in the docs that trying host passthrough could have success

Reply:
Well, passing `-cpu host` means all the supported instruction sets also gets passed through instead of manually specifying them. The CPUID vendor emulation is already done by QEMU (`vendor=GenuineIntel`) and by OpenCore (https://github.com/kholia/OSX-KVM/blob/master/OpenCore-Catalina/config.plist#L360). For certain AMD systems, the leaf7 patch becomes important but afaik it doesn't interfere with Intel systems. OpenCore already forces the kernel to pretend it's a Penryn family chip, https://github.com/kholia/OSX-KVM/blob/master/OpenCore-Catalina/config.plist#L408

Reply:
I wonder what @thenickdude has to say on this topic. Thanks!

Reply:
We don't have a kernel patch to force the CPU to look like Penryn for Big Sur yet. But yeah cpu=host works with Catalina, for Intel CPUs only, if OpenCore has the patch enabled.

Doesn't work for AMD CPUs

Reply:
@thenickdude my experience was a bit different, I was able to get (this is on Mojave though) the VM to work with `cpu=host` on AMD without patches (albeit without AVX and a lot of other instruction sets, for that I needed the `leaf7` patch). I think my configuration included `vendor=GenuineIntel` and OpenCore Cascade Lake emulation with a supported topology. 

Nested virtualization doesn't work on AMD though :(

Reply:
I recommend adding these CPU + associated config.plist changes to boot-passthrough.sh or a new script. Create a new folder for the new config.plist. Let's leave the rest of the working scripts as they are.

Reply:
Makes sense, I mean scope of doing changes in a passthrough setup is more tedious and we might end up breaking certain VMs with the other scripts, maybe a separate OpenCore directory `OC-Catalina-Passthrough` for `boot-passthrough.sh` I think should be sufficient. Does that sound reasonable?

Reply:
Yep, +1 for the `OC-Catalina-Passthrough` option. We can also modernize that super old `boot-passthrough.sh` script while at it.

Reply:
> Doesn't work for AMD CPUs without additional patching
AMD Threadripper 3970X CPU here and do not need a kernel patch needed for AMD CPUs to make it look like Penryn when using host cpu setting in libvirt xml. 
![Screen Shot 2020-08-29 at 10 55 46 AM](https://user-images.githubusercontent.com/33814640/91639670-38a04980-e9e6-11ea-9545-682f9c0d6ca8.png)
Only needs topology patch and leaf7 patch. 
![Screen Shot 2020-08-29 at 10 52 36 AM](https://user-images.githubusercontent.com/33814640/91639613-e101de00-e9e5-11ea-948a-138405dada41.png)
Also allows using OpenCore's Emulate CPU section like it should be used.
![Screen Shot 2020-08-29 at 10 53 42 AM](https://user-images.githubusercontent.com/33814640/91639623-ebbc7300-e9e5-11ea-9884-ccb535501a45.png)